pkg/controller/employee/update_employee_status.go

```diff
@@ -122,9 +125,21 @@ func (r *controller) removeDiscordRoles(discordUserID string) error {
 		}
 	}
 
-	// Assign alumni role
-	alumniRole := roles.ByCode("alumni")
-	err = r.service.Discord.AddRole(discordUserID, alumniRole.ID)
+	// Assign role based on IsKeepFwdEmail
+	var targetRole string
+	if e.IsKeepFwdEmail {
+		targetRole = "veteran"
+	} else {
+		targetRole = "alumni"
+	}
```

Comment:
can we use role_id for this, since there is chance we will rename the role name 🤔

Reply:
r e